### ISSUE TITLE
[1.2.0] NoClassDefFoundError crash on dedicated server during preInit

### ISSUE THREAD

**Issue by TigerWalts** [Invadermonky/VillagerContracts#2]
```
Description: Exception in server tick loop

net.minecraftforge.fml.common.LoaderExceptionModCrash: Caught exception from Villager Contracts (villagercontracts)
Caused by: java.lang.NoClassDefFoundError: net/minecraftforge/client/event/GuiScreenEvent$KeyboardInputEvent$Pre
    at java.lang.Class.getDeclaredMethods0(Native Method)
    at java.lang.Class.privateGetDeclaredMethods(Class.java:2701)
    at java.lang.Class.privateGetPublicMethods(Class.java:2902)
    at java.lang.Class.getMethods(Class.java:1615)
    at net.minecraftforge.fml.common.eventhandler.EventBus.register(EventBus.java:82)
    at com.invadermonky.villagercontracts.proxy.CommonProxy.preInit(CommonProxy.java:11)
    at com.invadermonky.villagercontracts.VillagerContracts.preInit(VillagerContracts.java:36)
```

This is caused by the `onItemRename()` handler:

https://github.com/Invadermonky/VillagerContracts/blob/239673ed8f9060a88ceeb872f242048f64a5cad4/src/main/java/com/invadermonky/villagercontracts/handlers/EventHandler.java#L69-L83

**Comment by Invadermonky** [comment]
~~Can you post a full crash log here? That is a forge event so the only reason it would be missing was if there was an issue with the install or something else strange going on.~~

Nevermind, I am able to reproduce the crash.

**Comment by Invadermonky** [comment]
Fixed with version [1.12.2-1.2.1](https://www.curseforge.com/minecraft/mc-mods/villager-contracts/files/5481496).
